config/routes.rb

```diff
@@ -3,6 +3,9 @@
   root to: 'static#index'
   scope 'api', defaults: { format: :json } do
     get '/auth/:provider/callback' => 'sessions#create'
+    if Rails.env.development?
```

Comment:
I believe this route is redundant with the one above it. It needs to exist regardless of the mode of the application.